documentation/source/users/cantherm/input.rst

```diff
@@ -33,61 +34,75 @@ Function                    Description
+``species()`` function the type and number of bonds for CanTherm to apply these corrections.
 The example below specifies CBS-QB3 as the model chemistry::
 
-    modelChemistry("CBS-QB3")
-
-Currently, atomization energy corrections (AEC), bond corrections (BC), and spin orbit correction (SOC) are available for the following model chemistries:
-
-================================================ ===== ==== ====
-Model Chemistry                                  AEC   BC   SOC     
-================================================ ===== ==== ====
-``'CBS-QB3'``                                     v    v    v
-``'G3'``                                          v         v
-``'M08SO/MG3S*'``                                 v         v
-``'M06-2X/cc-pVTZ'``                              v         v
-``'Klip_1'``                                      v         v
-``'Klip_2'`` *uses QCI(tz,qz) values*             v         v
-``'Klip_3'`` *uses QCI(dz,qz) values*             v         v
-``'Klip_2_cc'`` *uses CCSD(T)(tz,qz) values*      v         v
-``'CCSD-F12/cc-pVDZ-F12'``                        v         v
-``'CCSD(T)-F12/cc-pVDZ-F12_H-TZ'``                v         v
-``'CCSD(T)-F12/cc-pVDZ-F12_H-QZ'``                v         v
-``'CCSD(T)-F12/cc-pVnZ-F12'``, *n = D,T,Q*        v    v    v
-``'CCSD(T)-F12/cc-pVDZ-F12_noscale'``             v         v
-``'CCSD(T)-F12/cc-pCVnZ-F12'``, *n = D,T,Q*       v         v
-``'CCSD(T)-F12/aug-cc-pVnZ-F12'``, *n = D,T,Q*    v         v
-``'B-CCSD(T)-F12/cc-pVnZ-F12'``, *n = D,T,Q*      v         v
-``'B-CCSD(T)-F12/cc-pCVnZ-F12'``, *n = D,T,Q*     v         v
-``'B-CCSD(T)-F12/aug-cc-pVnZ-F12'``, *n = D,T,Q*  v         v
-``'DFT_G03_b3lyp'``                               v    v    v
-``'DFT_ks_b3lyp'``                                          v
-``'DFT_uks_b3lyp'``                                         v
-``'G03_PBEPBE_6-311++g_d_p'``                     v         v
-``'MP2_rmp2_pVnZ'``, *n = D,T,Q*                  v         v
-``'FCI/cc-pVnZ'``, *n = D,T,Q*                    v         v
-``'BMK/cbsb7'``                                   v    v    v
-``'BMK/6-311G(2d,d,p)'``                          v    v    v
+    modelChemistry = "CBS-QB3"
+
+Alternatively, the atomic energies at the ``modelChemistry`` level of theory can be directly
+specified in the input file by providing a dictionary of these energies in the following format::
+
+    atomEnergies = {
+        'H': -0.499818,
+        'C': -37.78552,
+        'N': -54.520543,
+        'O': -74.987979,
+        'S': -397.658253,
+    }
+
+The table below shows which model chemistries have atomization energy corrections (AEC), bond
+corrections (BC), and spin orbit corrections (SOC). It also lists which atoms types are available
+for a given model chemistry.
+
+================================================ ===== ==== ==== ==================
+Model Chemistry                                  AEC   BC   SOC  Atom Types
```

Comment:
I love this addition!
Could you just change "Atom Types" to something like "Supported elements"? "Atom Types" means something else in RMG.

Reply:
Sure thing! Too much RMG lingo...

Reply:
Done